### ISSUE TITLE
Allow to enable subtitles and speech at the same time

### ISSUE THREAD

**Issue by neuromancer** [neuromancer/wc2-re#14]
The speech pack allowed to have some voiced cutscenes, but removed the subtitles that are useful for non native speakers. Unfortunately, if just enabled, the subtitle tend to disgree with what the characters say, some phrases are lost, missing or changed, so we must find a way to restore it.

**Comment by schlangz** [comment]
That's common across any Origin game: the subtitles are not necessarily 100% the same compared to what the characters actually speak

**Comment by neuromancer** [comment]
Actually, the subtitles look correct, there was a bug affecting new lines as you pointed in #10. Anyway, this should be fixed already.
